### ISSUE TITLE
Indexing OCR fails with multibyte characters

### ISSUE THREAD

**Issue by patdunlavey** [esmero/strawberry_runners#48]
While testing for [this issue](https://github.com/esmero/strawberry_runners/issues/46), I found that OCR from some pages was failing to index in solr. As best I can tell/guess, this happens when some OCR with multi-byte characters is serialized and stored in the key_value table, and then fails to unserialize because the character counts in the serialized data don't match the byte counts in that data - or something like that.

It seems weird that something would serialize in a way that can't be unserialized! 

Anyway, I'm attaching a couple files. One is a pdf that resulted in the OCR with multibyte characters. The second is the text that I see from the first page in the keystore.

[CT_news_1871-01.pdf](https://github.com/esmero/strawberry_runners/files/8217375/CT_news_1871-01.pdf)
[western-ct-news-ocr-page-1.txt](https://github.com/esmero/strawberry_runners/files/8217385/western-ct-news-ocr-page-1.txt)

**Comment by DiegoPino** [comment]
Ok, cool. will check and let you know. I'm the BEST serializer/deserializer in this apartment so who know what I can find and how this can be fixed!

**Comment by patdunlavey** [comment]
I dare say you may be the BEST serializer/deserializer on your block!

**Comment by patdunlavey** [comment]
@DiegoPino I figured it out! I had checked out my [ISSUE-46-OCR](https://github.com/esmero/strawberry_runners/pull/47) branch from main, not recognizing that "strawberryfield/strawberry_runners": "0.2.0.x-dev" actually fetches the 0.3.0 branch - which has not been merged to main. As a result, I did not have this fix in my issue branch. When I merged 0.3.0 to my issue branch locally, solr indexing of the OCR completes without complaint.

Two questions:
1. Shall I go ahead and merge 0.3.0 into my issue branch?
2. Shall I change the target branch of the PR to 0.3.0?
Thanks!
